data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3482 @@
+					goto ship
+				`	"If this world is so wild, why put the jump drives here in the first place?"`
+			`	"The Unfettered are not the ones who placed those jump drives here..." He really does not seem to want to tell you much about this vault.`
+			
+			label ship
+			`	You go back to your ship with the Hai, and give the orders to collect and install the jump drives.`
+			`	A bit later the Hai comes to speak to you on the bridge. "It took us too long to install the jump drives, and we seem to have attracted attention. I guess that if some of the unfettered humans got away from the vault, they have contacted some friends..." As you look outside you see multiple ships getting ready to launch far away. You launch as soon as you can, rushing trough the correct procedures.`
+				launch
+	
+	on accept
+		"outfit" "Jump Drive" 4
+	npc evade
+		government "Pirate"
+		personality disables plunders vindictive
+		system "Almaaz"
+		fleet "Large Northern Pirates"
+		fleet "Small Northern Pirates" 2
+
+
+
+fleet "Marauders"
+	government "Mercenaries"
+	names "pirate"
+	personality
+		disables plunders harvests
+	variant 2
+		"Marauder Firebird (Engines)"
+	variant 3
+		"Marauder Firebird"
+	variant 3
+		"Marauder Arrow (Weapons)"
+		"Marauder Bounder"
+	variant 3
+		"Marauder Falcon"
+	variant 3
+		"Marauder Raven (Weapons)"
+	variant 1
+		"Marauder Raven"
+		"Marauder Quicksilver (Weapons)" 2
+	variant 3
+		"Marauder Splinter (Weapons)"
+	variant 2
+		"Marauder Manta" 2
+	variant 1
+		"Marauder Leviathan (Engines)"
+	variant 1
+		"Marauder Raven" 2
+		
+
+mission "Unfettered: Nemo Me Impune Lacessit"
+	invisible
+	repeat
+	landing
+	minor
+	source
+		near "Ya Hai" 10
+	to offer
+		has "Unfettered: Jump Drive Retrieval: done"
+		not "sold bars"
+		random < 3
+	on offer
+		require "Gold Bars"
+	npc disable
+		government "Mercenaries"
+		system
+			distance 2 5
+		fleet "Marauders" 3
+		dialog `After disabling the last of the pirate ships, you wonder what could have motivated such an attack. Maybe you possess something valuable to them? When they entered the system they only sent one message: "nemo me impune lacessit". Whatever it is they were up against, it is probably best to store it on a planet (using U) or get rid of it.`
+
+mission "Unfettered: Nemo Me Impune Lacessit 2"
+	invisible
+	repeat
+	landing
+	minor
+	source
+		near Sol 100
+	to offer
+		has "Unfettered: Jump Drive Retrieval: done"
+		not "sold bars"
+		random < 1
+	on offer
+		require "Gold Bars"
+	npc disable
+		government "Mercenaries"
+		system
+			distance 2 5
+		fleet "Marauders" 5
+		dialog `After disabling the last of the pirate ships, you wonder what could have motivated such an attack. Maybe you possess something valuable to them? When they entered the system they only sent one message: "nemo me impune lacessit". Whatever it is they were up against, it is probably best to store it on a planet (using U) or get rid of it.`
+
+
+
+mission "Unfettered: I Shall Go Unpunished"
```

Comment:
From here on there's a lot of this, all of these empty line gaps inside missions shouldn't be there.

Reply:
I used those to separate "sections" of the mission for clarity but I can remove them